Kernel/Firmware/ACPI/Parser.cpp

```diff
@@ -95,7 +95,8 @@ UNMAP_AFTER_INIT void ACPISysFSDirectory::find_tables_and_register_them_as_compo
 UNMAP_AFTER_INIT NonnullRefPtr<ACPISysFSDirectory> ACPISysFSDirectory::must_create(FirmwareSysFSDirectory& firmware_directory)
 {
     auto acpi_directory = MUST(adopt_nonnull_ref_or_enomem(new (nothrow) ACPISysFSDirectory(firmware_directory)));
-    acpi_directory->find_tables_and_register_them_as_components();
+    if (ACPI::is_enabled())
```

Comment:
I'd argue to not create the folder entirely as it has no use if ACPI is disabled.

Reply:
Yeah, I wasn't sure which is preferred, since as far as userspace programs are concerned, an ACPI without any tables is the same as no ACPI whatsoever no? (it means you don't have to write code in userspace that first checks if the folder exists)

Reply:
sure, will just not add the folder then